src/OpalCompiler-Core/CompilationContext.class.st

```diff
@@ -699,6 +699,24 @@ CompilationContext >> requestor: anObject [
 	requestor := anObject
 ]
 
+{ #category : #'as yet unclassified' }
```

Comment:
The ci will probably not like the unclassified method

Reply:
indeed